apps/src/applab/designMode.js

```diff
@@ -188,7 +203,7 @@ designMode.onPropertyChange = function(element, name, value) {
     default:
       throw "unknown property name " + name;
   }
-  Applab.levelHtml = designMode.serializeToLevelHtml();
+  designMode.serializeToLevelHtml();
```

Comment:
it's worth noting that we are now making all elements undraggable/unresizable and then draggable/resizable on every keystroke within the property editor.